models/artifact_model.go

```diff
@@ -88,6 +88,7 @@ type Artifact struct {
 	ServiceID  string       `reform:"service_id"`
 	DataModel  DataModel    `reform:"data_model"`
 	Status     BackupStatus `reform:"status"`
+	ScheduleID string       `reform:"schedule_id"`
```

Comment:
Why do we need it here? We have this relation `scheduled task` -> `backup job` -> `artifact`, why it's not enough?

Reply:
Because of retention.
We need to delete old artifacts (at some time). And I think it's good to know which artifacts were made by which scheduled backup.
Or I'm missing something and we can do it, even without this field?

Reply:
Well, I don't know for now what approach is better, maybe we can just set some TTL for artifacts, maybe something else. But anyway, retention is out of the scope of this ticket, so let's think later about it. And for now, let's remove this field if retention is the only reason to have it.

Reply:
I also think it would be good to keep this field (later we could show on the UI what scheduled backup created this artifact).

Reply:
But artifact hasn't JobID for example, and scheduler even more high-level entity.

Reply:
Now links dependency looks like:
Job has a link to Artifact, Artifact has a link to Scheduled Task, Scheduled Task triggers Job, Job created Artifact.

Reply:
Maybe Job should have a link to Scheduled Task?

Reply:
Resolved after discussion.